torch/_logging/_internal.py

```diff
@@ -179,6 +179,7 @@ def clear(self):
     "dynamo": logging.DEBUG,
     "aot": logging.DEBUG,
     "inductor": logging.DEBUG,
+    "fsdp": logging.DEBUG,
```

Comment:
Is this setting default to debug? Is the default value for `TORCH_LOGS=""`, or `TORCH_LOGS=fsdp`?